Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -59,7 +59,7 @@ def parser(cls, parser, loggers=None):
         )
         parser.add_argument(
             '--pr', '--no-pr',
-            default=True,
+            default=False,
```

Comment:
Weird to have this `False` is `--no-pr` is still an option. Should we maybe make it `None`? Feels like `--no-pr` + `--safe` should be an error, those arguments don't make sense together.